pkg/openapi-gen/openapi-gen.go

```diff
@@ -342,6 +342,9 @@ func buildSchema(deviceEntry *yang.Entry, parentState yang.TriState, parentPath
 			default:
 				return nil, nil, fmt.Errorf("unhandled leaf %v %s", dirEntry.Type.Kind, dirEntry.Type.Name)
 			}
+			if !dirEntry.Config.Value() {
```

Comment:
value of Config is yang.TriState - here you're only checking it's not 0 (TSUnset). 
TSUnset will mean it will take the value of its parent or TSTrue. 
TSFalse will mean it's readonly.
If an items parent is TSFalse, then everything beneath it is TSFalse

Reply:
@SeanCondon I checked it and it didn't handle the parent, now I'm using [Entry.ReadOnly()](https://pkg.go.dev/github.com/openconfig/goyang/pkg/yang@v1.1.0#Entry.ReadOnly) which checks the parent's state if it's unset.
I added two tests, one when parent is configurable, but one of leaves is not, and the other when parent is un-configurable, but leaves are set true.